src/index.html

```diff
@@ -1,12 +1,426 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="/">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+             class="nav__item-link is-active"
+             href="/apple"
+            >
+             Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/samsung">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/smartphones">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-link"
+              href="/laptops"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/gadgets">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/tablets">
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/photo">
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/video">
+              Video
+            </a>
+          </li>
+        </ul>
+
+      </nav>
+    </header>
+    <main>
+      <section class="catalog">
+        <div class="catalog__container">
+          <article class="card catalog__card" data-qa="card">
+            <a class="card__link">
+              <img
+                class="card__link-img"
+                src="images/imac.jpeg"
+                alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+              >
+            </a>
+            <h3 href="/" class="card__name">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h3>
+
+            <p class="card__code"> Product code: 195434 </p>
+            <div class="card__rating">
+              <ul class="stars stars--4">
+                <li class="stars__star"></li>
+
+                <li class="stars__star"></li>
+
+                <li class="stars__star"></li>
+
+                <li class="stars__star"></li>
+
+                <li class="stars__star"></li>
+              </ul>
+
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-title">Price</p>
+
+              <p class="card__price-current">$2,199</p>
+            </div>
+            <div class="card__action">
```

Comment:
Do you need this extra wrapper here? Can we move all styles to `card__buy-button`?

Reply:
I see, of course, this block must be deleted